Makefile

```diff
@@ -231,6 +231,17 @@ push-images:
 	./mage build:pushimages
 .PHONY: push-images
 
+## Build and push only the addon-operator-package
+build-push-aopackage:
```

Comment:
As it's a Makefile target in the ADO repository it's obvious we're building the operator package thus no need to explicitly mention the operator name.

```suggestion
build-push-package:
```